docs/documentation/server_admin/topics/roles-groups/proc-managing-groups.adoc

```diff
@@ -12,6 +12,15 @@ Groups are hierarchical. A group can have multiple subgroups but a group can hav
 
 If you have a parent group and a child group, and a user that belongs only to the child group, the user in the child group inherits the attributes and role mappings of both the parent group and the child group.
 
+The hierarchy of a group is sometimes represented using the group path. The path is the complete list of names that represents the hierarchy of a specific group, from top to bottom and separated by slashes `/` (similar to files in a File System). For example a path can be `/top/level1/level2` which means that `top` is a top level group and is parent of `level1`, which in turn is parent of `level2`. This path represents unambiguously the hierarchy for the group `level2`.
+
+Because of historical reasons {project_name} does not escape slashes in the group name itself. Therefore a group named `level1/group` under `top` uses the path `/top/level1/group`, which is misleading. {project_name} can be started with the option `--spi-group-jpa-escape-slashes-in-group-path` to `true` and then the slashes in the name are escaped with the character `~`. The escape char marks the slash is part of the name and has no hierarchical meaning. The previous path example would be `/top/level1~/group` when escaped.
```

Comment:
```suggestion
Because of historical reasons {project_name}, does not escape slashes in the group name itself. Therefore a group named `level1/group` under `top` uses the path `/top/level1/group`, which is misleading. {project_name} can be started with the option `--spi-group-jpa-escape-slashes-in-group-path` to `true` and then the slashes in the name are escaped with the character `~`. The escape char that marks the slash is part of the name and has no hierarchical meaning. The previous path example would be `/top/level1~/group` when escaped.
```

Reply:
@andymunro I think that you have understood this one wrong, the escape char is the one that marks, I mean, the escape char points out that the slash is part of the name. So maybe `The escape chars marks that the slash is part of the name...`, right?

Reply:
Okay, that works better. Sounds good.